### PR TITLE
Remove athletic team

### PR DESCRIPTION
**Pull request by teryhill**
removing the athletic team code

### CODE REVIEW

**Comment by bradymiller** [comment]
Hi Terry,
Announce a request to review this on the forums. I am guessing that Rod will interested in looking at this.
-brady

File: interface/forms/newpatient/common.php [hunk 1]
```diff
@@ -307,18 +303,6 @@ class="css_button link_submit" target='Main' onClick="top.restoreSession()">
...
-    <tr>
-     <td class='text' colspan='2' style='padding-top:1em'>
-<?php if ($GLOBALS['athletic_team']) { ?>
-      <p><i>Click [Add Issue] to add a new issue if:<br />
-      New injury likely to miss &gt; 1 day<br />
-      New significant illness/medical<br />
-      New allergy - only if nil exist</i></p>
-<?php } ?>
-     </td>
-    </tr>
```
Comment (bradymiller) [inline interface/forms/newpatient/common.php:320]: Recommend testing and making sure removing the <tr></tr> doesn't mess anything up (agree with removing it, just ensure test it out).

**Comment by bradymiller** [comment]
Also should remove this:
https://github.com/openemr/openemr/blob/master/sql/database.sql#L2597-L2600

**Comment by bradymiller** [comment]
Remove:
https://github.com/openemr/openemr/blob/master/library/lists.inc#L43

**Comment by bradymiller** [comment]
Remove:
https://github.com/openemr/openemr/blob/master/library/lists.inc#L63-L65

**Comment by bradymiller** [comment]
Remove:
https://github.com/openemr/openemr/blob/master/templates/report/general_default.html#L148-L158

**Comment by bradymiller** [comment]
Remove:
https://github.com/openemr/openemr/blob/master/contrib/forms/xmlformgen/xslt/common_objects.xslt#L163-L170

**Comment by bradymiller** [comment]
Review complete.

**Comment by teryhill** [comment]
removed

```
  From: Brady Miller <notifications@github.com>
```

 To: teryhill/openemr openemr@noreply.github.com 
Cc: Terry Hill teryhill@yahoo.com; Author author@noreply.github.com
 Sent: Tuesday, May 10, 2016 2:42 AM
 Subject: Re: [teryhill/openemr] Remove athletic team (#4)

Also should remove this:
https://github.com/openemr/openemr/blob/master/sql/database.sql#L2597-L2600—

**Comment by teryhill** [comment]
removed

```
  From: Brady Miller <notifications@github.com>
```

 To: teryhill/openemr openemr@noreply.github.com 
Cc: Terry Hill teryhill@yahoo.com; Author author@noreply.github.com
 Sent: Tuesday, May 10, 2016 2:45 AM
 Subject: Re: [teryhill/openemr] Remove athletic team (#4)

Remove:
https://github.com/openemr/openemr/blob/master/library/lists.inc#L43—

**Comment by teryhill** [comment]
removed

```
  From: Brady Miller <notifications@github.com>
```

 To: teryhill/openemr openemr@noreply.github.com 
Cc: Terry Hill teryhill@yahoo.com; Author author@noreply.github.com
 Sent: Tuesday, May 10, 2016 2:45 AM
 Subject: Re: [teryhill/openemr] Remove athletic team (#4)

Remove:
https://github.com/openemr/openemr/blob/master/library/lists.inc#L63-L65—

**Comment by teryhill** [comment]
removed

```
  From: Brady Miller <notifications@github.com>
```

 To: teryhill/openemr openemr@noreply.github.com 
Cc: Terry Hill teryhill@yahoo.com; Author author@noreply.github.com
 Sent: Tuesday, May 10, 2016 3:16 AM
 Subject: Re: [teryhill/openemr] Remove athletic team (#4)

Remove:
https://github.com/openemr/openemr/blob/master/templates/report/general_default.html#L148-L158—

**Comment by teryhill** [comment]
removed

```
  From: Brady Miller <notifications@github.com>
```

 To: teryhill/openemr openemr@noreply.github.com 
Cc: Terry Hill teryhill@yahoo.com; Author author@noreply.github.com
 Sent: Tuesday, May 10, 2016 3:18 AM
 Subject: Re: [teryhill/openemr] Remove athletic team (#4)

Remove:
https://github.com/openemr/openemr/blob/master/contrib/forms/xmlformgen/xslt/common_objects.xslt#L163-L170—

File: interface/forms/newpatient/common.php (hunk 1 above)
Comment (sunsetsystems) [inline interface/forms/newpatient/common.php:320]: Yes, be very careful about removing any code not wrapped in this test.  There was surely a reason it was done that way.  Note the colspan.

File: interface/patient_file/encounter/forms.php [hunk 2]
```diff
@@ -580,7 +580,7 @@ function divtoggle(spanid, divid) {
 }
 ?>
 
-<?php if ($GLOBALS['athletic_team'] && $GLOBALS['concurrent_layout'] == 2) { ?>
```
Comment (sunsetsystems) [inline interface/patient_file/encounter/forms.php:583]: No, this is && not ||.  The code block should be removed.

File: interface/main/left_nav.php [hunk 3]
```diff
@@ -725,55 +719,6 @@ function reloadEncounter(frname) {
   }
  }
 
- // Clear and reload issue-related menu items for active_pid.
- // Currently this only applies to athletic teams, but might be implemented
```
Comment (sunsetsystems) [inline interface/main/left_nav.php:729]: Given the comment I would leave these functions here but comment them out, as they may be of interest later.

File: interface/forms/newpatient/common.php (hunk 1 above)
Comment (teryhill) [inline interface/forms/newpatient/common.php:320]: added back the tr's and td's with the colspan

File: interface/main/left_nav.php (hunk 3 above)
Comment (teryhill) [inline interface/main/left_nav.php:729]: added back and commented out

File: interface/patient_file/encounter/forms.php (hunk 2 above)
Comment (teryhill) [inline interface/patient_file/encounter/forms.php:583]: removed

**Comment by bradymiller** [comment]
I just committed this to the codebase.

**Pull request closed without merging**